### ISSUE TITLE
Missing user notification of dynamic search

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3117]
When a user starts typing into the search field, the displayed datasets are updated  dynamically to reflect the search input. Screen reader users are not notified of the  dynamically rendered datasets and when nothing matches the search query, the text  "nothing found. Try another search!" is displayed but is not announced to screen reader  users. 

**Suggested Solution** 
- Notify users of the dynamically updated datasets by providing off-screen  instructions, e.g. "Datasets will update as you type", and use the aria-describedby attribute on the search input referencing the ID of the instructions container to  announce them when the field is focused.  
- Implement the status message "nothing found. Try another search!" as an assertive  live region to be announced once it's displayed. Note that the <div> with aria live="assertive" and aria-atomic="true" should be present in the DOM when the  page is loaded, and the text should be dynamically added when no results are  available. 

Severity: 2
